pkg/legacy/converter.go

```diff
@@ -217,3 +218,33 @@ func buildConfigProviders(agentConfig Config) ([]config.ConfigurationProviders,
 
 	return []config.ConfigurationProviders{cp}, nil
 }
+
+func buildHistogramAggregates(agentConfig Config) []string {
+	configValue := agentConfig["histogram_aggregates"]
+
+	var histogramBuild []string
+	var result []string
+	validValues := [6]string{"min", "max", "median", "avg", "sum", "count"}
+
+	if configValue == "" {
+		// this is expected, not an error
+		return nil
+	}
+
+	result = strings.Split(configValue, ", ")
```

Comment:
people might not have split their values with a `, ` but just by a `,`. We should first remove every space and then split by comas.